services/blockchain-connector/shared/sdk/nft.js

```diff
@@ -37,6 +41,46 @@ const getNFTConstants = async () => {
 	}
 };
 
+// TODO: Remove lint suppression once sdk endpoint is available
+// eslint-disable-next-line no-unused-vars
+const updateCollectionIDs = async (params) => {
+	try {
+		// TODO: Invoke sdk endpoint once available
+		// moduleConstants = await invokeEndpoint('nft_getCollectionIDs',{ chainID: params.chainID });
```

Comment:
Incorrect endpoint. From the current implementation (LS and SDK), looks like it'll only return the collectionIDs for the current chain's collections existing in the NFT sub-store.

This information should actually come from a different endpoint, similar to `token_getSupportedTokens`. Please discuss with the SDK team and create an issue in their repo, if necessary 🙏

Reply:
Applied. related sdk issue: https://github.com/LiskHQ/lisk-sdk/issues/8886